classes/Cart.php

```diff
@@ -3233,27 +3233,29 @@ public function getTotalShippingCost($delivery_option = null, $use_tax = true, C
         if (isset(Context::getContext()->cookie->id_country)) {
             $default_country = new Country(Context::getContext()->cookie->id_country);
         }
-        if (null === $delivery_option) {
+        if (is_null($delivery_option)) {
             $delivery_option = $this->getDeliveryOption($default_country, false, false);
         }
 
-        $_total_shipping = array(
-            'with_tax' => 0,
-            'without_tax' => 0,
-        );
         $delivery_option_list = $this->getDeliveryOptionList($default_country);
+
+        $_total_shipping = 0.0;
         foreach ($delivery_option as $id_address => $key) {
+
             if (!isset($delivery_option_list[$id_address]) || !isset($delivery_option_list[$id_address][$key])) {
                 continue;
             }
 
-            $_total_shipping['with_tax'] += $delivery_option_list[$id_address][$key]['total_price_with_tax'];
-            $_total_shipping['without_tax'] += $delivery_option_list[$id_address][$key]['total_price_without_tax'];
+            if ($use_tax) {
+                $_total_shipping += $delivery_option_list[$id_address][$key]['total_price_with_tax'];
+            } else {
+                $_total_shipping += $delivery_option_list[$id_address][$key]['total_price_without_tax'];
+            }
         }
-
-        return ($use_tax) ? $_total_shipping['with_tax'] : $_total_shipping['without_tax'];
+        return $_total_shipping;
```

Comment:
An additional line is required to make the lint job pass.

```suggestion

        return $_total_shipping;
```